erpnext/setup/doctype/item_group/item_group.py

```diff
@@ -37,30 +37,34 @@ def validate(self):
 	def on_update(self):
 		NestedSet.on_update(self)
 		invalidate_cache_for(self)
+		self.make_route()
 		self.validate_name_with_item()
 		self.validate_one_root()
 		self.delete_child_item_groups_key()
 
 	def make_route(self):
 		'''Make website route'''
-		if not self.route:
-			self.route = ''
-			if self.parent_item_group:
-				parent_item_group = frappe.get_doc('Item Group', self.parent_item_group)
+		self.route = ''
+		if self.parent_item_group:
+			parent_item_group = frappe.get_doc('Item Group', self.parent_item_group)
 
-				# make parent route only if not root
-				if parent_item_group.parent_item_group and parent_item_group.route:
-					self.route = parent_item_group.route + '/'
+			# make parent route only if not root
+			if parent_item_group.parent_item_group and parent_item_group.route:
+				self.route = parent_item_group.route + '/'
 
-			self.route += self.scrub(self.item_group_name)
+		self.route += self.scrub(self.item_group_name)
 
-			return self.route
+		return self.route
 
 	def on_trash(self):
 		NestedSet.on_trash(self)
 		WebsiteGenerator.on_trash(self)
 		self.delete_child_item_groups_key()
 
+	def after_rename(self, old_name, new_name, merge):
+		self.make_route()
+		self.save()
```

Comment:
frappe.db.set_value("Item Group", self.item_group, "route", self.route)

Reply:
Done. But no `item_group` attribute found, it should be `item_group_name`.